tools/python/boutiques/bosh.py

```diff
@@ -358,27 +363,64 @@ def test(*params):
     return pytest.main([test_path, "--descriptor", result.descriptor])
 
 
+def search(*params):
+    parser = ArgumentParser("Search Zenodo for Boutiques descriptors. "
+                            "When no term is supplied, will search for "
+                            "all descriptors.")
+
+    parser.add_argument("-q", "--query", action="store", help="Search query")
```

Comment:
I would make it a positional argument.

Reply:
Ok, but that would make it mandatory, right? I made it optional so that you could just type `bosh search` and see all the descriptors.

Reply:
nope, there can be optional positional arguments: https://stackoverflow.com/questions/4480075/argparse-optional-positional-arguments